### ISSUE TITLE
how to config with python3

### ISSUE THREAD

**Issue by zcanfly** [chaquo/chaquopy-hello#1]
how to config with python3?I change `build.gradle`setting with 
```
python {
             buildPython "/Library/Frameworks/Python.framework/Versions/3.5/bin/python3.5"
            version "3.5.4"
            staticProxy "hello"
        }
```

but android studio refresh error with `Error:Cause: debug: invalid Python version '3.5.4'. Available versions are [2.7.10].`

**Comment by zcanfly** [comment]
version "3.6.3" is work well.

**Comment by mhsmith** [comment]
Yes, Python 3.6.3 is available in Chaquopy 1.1.0. I'll update this project to match.
